### DISCUSSION TITLE
2 Sided Bar Chart

### DISCUSSION THREAD

**Discussion by scottquested** [ChartsCSS/charts.css#69] [Q&A]
Do you have any plans to implement a 2 sided bar chart, something similar to the below?

![B6745DC8-0C61-4822-9779-1229D38D0B4B](https://user-images.githubusercontent.com/4488112/158014687-3a3daa8d-3f4c-49bc-b1a8-7a93d5fa75cf.jpeg)

If you haven’t I’d love to try and contribute to this.

**Comment by ramiy** [comment]
Hi @scottquested 

Version 1.0.0 implements something similar, by combining two bar charts with reversed labels side by side, one with regular the other with reversed orientation, you can get a beautiful [pyramid chart](http://localhost:8080/components/orientation/#population-pyramid-chart).
